controllers/admin/AdminShopUrlController.php

```diff
@@ -493,7 +493,7 @@ public function processUpdate()
     {
         $this->redirect_shop_url = false;
         $current_url = parse_url($_SERVER['REQUEST_URI']);
-        if (trim(dirname(dirname($current_url['path'])), '/') == trim($this->object->getBaseURI(), '/')) {
+        if (trim(dirname($current_url['path'],2), '/') == trim($this->object->getBaseURI(), '/')) {
```

Comment:
sorry , `level` params of `dirname` function was added on php 7.0;
http://php.net/manual/fr/function.dirname.php
i does not know if php 7 will be required for ps , can please ps team answer about the php minimum version ?

Reply:
Nice catch, thanks.